code/clement/django/lab03/django_pokedex/templates/pokedex/_filter-form-poked.html

```diff
@@ -0,0 +1,22 @@
+<div class="bg-dark">
+    <div class="col-12 col-md-3 mx-4 pt-3">
+        <form action="{% url 'pokedex_app:index' %}" method="POST" class="row gap-1">
+
+            <!-- {{ form_data }} -->
+
+            <!-- POKEMON SEARCH TEXT INPUT -->
+            <fieldset class="col-8 form-group">
+                <label for="pokemon-search" class="form-label">
+                    <h4 class="h4">Pokemon Search</h4>
+                    <input type="text" id="pokemon-search" name="pokemon-search" class="form-control">
+                </label>
+            </fieldset>
+
+            {% csrf_token %}
+
+            <div class="col-4 offset-4 d-flex justify-content-center mx-auto">
```

Comment:
`mx-auto` probably isn't necessary here, because of `d-flex justify-content-center`